pytorch_lightning/core/lightning.py

```diff
@@ -119,6 +119,20 @@ def optimizers(self, use_pl_optimizer: bool = True) -> Union[Optimizer, List[Opt
         # multiple opts
         return opts
 
+    def lr_schedulers(self) -> Optional[Union[Any, List[Any]]]:
+        if not self.trainer.lr_schedulers:
+            return None
+
+        # ignore other keys "interval", "frequency", etc.
+        lr_schedulers = [s["scheduler"] for s in self.trainer.lr_schedulers]
```

Comment:
Even when dict keys like `"interval"`, `"monitor"`, are defined in `configure_optimizers()`, this line ignores all of the keys except `"scheduler"`. [[lr scheduler docs](https://pytorch-lightning.readthedocs.io/en/latest/common/optimizers.html#learning-rate-scheduling)]